### ISSUE TITLE
Linux LV2: xruns and dropouts with [openpanel -s(

### ISSUE THREAD

**Issue by GModal** [pierreguillot/Camomile#197]
2. **Describe your environment**
Debian Buster, stable
Carla 2.2.0-RC1
Camomile 1.0.7 Beta1x (Linux 64 binary zip) (the beta version you provided)

3. **Describe the problem or the feature request**

Many xruns produced when the open file dialog is instantiated. Additionally, as the mouse pointer moves over the _Open..._ file list, more xruns & dropouts happen (while the dialog is open).
I'm using the [openpanel -s( message. The behavior is present with or without the -s flag.

Note: this behavior does **not** happen with the Linux VST3 plugin. Yes, there's a minor audio dropout while the file is loaded (again, using -s), but that's the expected behavior with -s. 

NO xruns with the VST3 plugin. Same source files for both plugins.

**Comment by pierreguillot** [comment]
Can you try again with camomile v1.0.8-beta4 and tell me if there is still an issue?

**Comment by GModal** [comment]
Tried it in Ardour with LV2 plugin, and it now works correctly.

However, I'm currently using a more capable PC w/ Ubuntu 20.04. I will check the openpanel behavior on the older Debian install tomorrow.

We may have another issue -- all my Camomile VST3 plugins are blacklisted on Ardour. I don't think that was the case previously. I will test this with builds of your examples, and VST3's from older versions (I will test tomorrow, and open a new issue if needed).

**Comment by pierreguillot** [comment]
Ok, thank you! It might be necessary to contact the Ardour support also.

**Comment by GModal** [comment]
It seems to be consistent behavior on more than one distro. Could this be a JUCE dependency problem?

**Comment by pierreguillot** [comment]
I get a bit lost here, can you resume what you tested (with the plugin versions, the DAW, the OS and the results)? And if you can share the plugin that you used to test and even your DAW session, it would be helpful.

**Comment by GModal** [comment]
Yeah, I'm lost too. Re: Ardour and VST3, correct? I guess I should open a new issue.

**Comment by pierreguillot** [comment]
No, please, do not open many issues for the same problem. I don't understand if this dropouts issue is fixed or not. If not, please give further information.

**Comment by pierreguillot** [comment]
I tested and it works
- Ubuntu 20.04.1 LTS - 64bit 
-  Ardour 6.5.0 & Carla 2.2.0
- PdStal plugin example and Camomile v1.0.8 beta4 

Can you test it and confirm?

**Comment by pierreguillot** [comment]
> Re: Ardour and VST3, correct? I guess I should open a new issue.

I don't think that there is a problem with the VST3 plugins but I think that Ardour doesn't support VST3 ;)

**Comment by pierreguillot** [comment]
I updated Juce and changed the way the open and save panels are handled, can you try these binaries and tell me if it's okay now? Thank you. 

https://github.com/pierreguillot/Camomile/actions/runs/1402491699
